### PR TITLE
phpunit5 support

### PR DESCRIPTION
**Pull request by ktamas77**
I added PHPUnit 5 support.
The Driver signature is the same as 6.

### CODE REVIEW

**Comment by krakjoe** [comment]
Just to confirm, has this actually been tested with a project that uses 5 ?

**Comment by ktamas77** [comment]
> Just to confirm, has this actually been tested with a project that uses 5 ?

Hello, yes I tested it locally (just sent a patch to match PHP 5 class names properly - good call).
Currently running on my local machine, works perfectly with PHPUnit 5.

File: bin/pcov [hunk 1]
```diff
@@ -71,16 +71,19 @@ if ($config["path"]) {
 	}
 }
 
-if (!\class_exists(\PHPUnit\Runner\Version::class)) {
-	usage("PHPUnit does not seem to be available");
+if (class_exists(PHPUnit_Runner_Version::class) &&
```
Comment (krakjoe) [inline bin/pcov:74]: I'm not sure why this is the first branch now ?

Comment (ktamas77) [reply bin/pcov:74]: Here https://github.com/krakjoe/pcov-clobber/pull/3/files#diff-81e693f3e3674a31805112e70f3c3843R77 you are testing for the existence for class `\PHPUnit\Runner\Version` - which exists only after PHPUNIT 6 or above.

If this class doesn't exists, the script will fail.

However in PHPUnit 5 the equivalent of this class is called `PHPUnit_Runner_Version`. I wanted to be sure that this class exists in conjunction with the right version number to determine the version, before we check on PHPUnit 6+ class syntax below, with minimal refactoring.

Comment (krakjoe) [reply bin/pcov:74]: Fair enough, I should have read it more closely :)

**Comment by krakjoe** [comment]
If you could just include a change for the readme, I think that's the last mention of versions anywhere in this repo ?

**Comment by ktamas77** [comment]
> If you could just include a change for the readme, I think that's the last mention of versions anywhere in this repo ?

readme updated - thank you for this amazing coverage tool (and the wrapper) it really saves a lots of time. i'm glad to be able to run with phpunit 5 as well now :)

**Pull request merged**